ui/validate_address.go

```diff
@@ -26,8 +28,10 @@ type validateAddressPage struct {
 	addressEditor         decredmaterial.Editor
 	clearBtn, validateBtn decredmaterial.Button
 	wallet                *wallet.Wallet
+	info                  *wallet.MultiWalletInfo
```

Comment:
There's no need to add this, you can get the wallet data directly from dcrlibwallet. Have a look at the suggested change below